### DISCUSSION TITLE
Making Unpoly More Attractive for Corporates

### DISCUSSION THREAD

**Discussion by kucharzyk** [unpoly/unpoly#664] [Ideas]
Hi All,

Unpoly is a great and powerful library, but it seems that not many companies have given it a try. Nowadays, libraries like htmx are gaining a lot of traction — this could be an opportunity for Unpoly as well.

I have some ideas on how we could improve Unpoly's visibility and attract more attention:

- Add a list of projects or companies that are using Unpoly.
- Highlight that Unpoly is maintained by more than one person (to address the bus factor concern).
- Create content for Twitter or YouTube to showcase where Unpoly shines.
- Offer commercial support to make corporates more comfortable using it.
- Participate in Hacktoberfest to encourage more contributions.

What do you think? Is there anything else we could do?

**Comment by azurha** [comment]
Hi o/
I'm a noob, and noobs like me will direct their learning towards the technologies that have the most noobfriendly tutorials/courses.
If there are more and more developers on the market, companies will adapt.

**Comment by triskweline** [comment]
Thanks for talking about this @kucharzyk .

> Add a list of projects or companies that are using Unpoly.

I [started a search for companies using Unpoly](https://github.com/unpoly/unpoly/discussions/666). Ideally we could turn this into a logo wall of popular brands on unpoly.com and in the README on GitHub.

> Highlight that Unpoly is maintained by more than one person (to address the bus factor concern).

We tried tagging trivial issues with [`good first issue`](https://github.com/unpoly/unpoly/issues?q=is%3Aissue+is%3Aclosed+label%3A%22good+first+issue%22) but this didn't improve MRs. For example [*Allow [up-emit] on buttons*](https://github.com/unpoly/unpoly/issues/416) could be fixed by [removing a single character](https://github.com/unpoly/unpoly/commit/fc9a321a8daa2e0ac5ead5b21c720fa592651e9e#diff-5f9bea7d68676170ed73619e6b508588e748c29fd0632b2bcf751c8e78c2d766R500) without needing to understand internals at all. However, the issue remained open for 2 years without an MR.

Frankly I made my peace with this. I'm super happy when people submit documentation fixes, participate in discussions and help out with issues. up. In particular @adam12 is a super hero for triaging issues while I'm busy, preparing reproductions, etc. 

> Create content for Twitter or YouTube to showcase where Unpoly shines.

htmx really took of when a YouTube influencer started making videos about it. Weirdly Unpoly's biggest marketing campaign is now Carson Gross (of htmx), who regularly tweets about other hypermedia libraries.

It would help a lot if more people could tweet about Unpoly, or posting a blog about your experiences from real projects (both good and bad).

If you like a podcast or follow a tech YouTuber, ask them to make an episode about Unpoly. Happy to join as a guest if that helps..

> Offer commercial support to make corporates more comfortable using it.

There will be a commercial support offering in the next weeks. Seeing some income there would also allow me to spend more time on Unpoly.
